packages/schema/src/Schema.ts

```diff
@@ -4921,10 +4921,23 @@ const makeClass = <A, I, R>(
 
     static get ast() {
       const toSchema = to(selfSchema)
+      const encode = Parser.encode<any, any, never>(toSchema)
```

Comment:
I think that this is `Parser.encodeUnknown`

Reply:
👍🏻